editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/impl/simple/AMQTemplate.java

```diff
@@ -36,6 +39,11 @@ public boolean supportsDSL(CamelDSLType type) {
 		}	
 	}
 	
+	@Override
+	public boolean isCompatible(String camelVersion) {
+		return new ComparableVersion(camelVersion).compareTo(maximalCompatibleCamelVersion) < 0;
```

Comment:
So is there at maximum only one comparable version for a template? What if it doesn't change between versions?

Reply:
only one version is compared at a time which is the version selected by user in the previous Wizard Page.